### PR TITLE
route new connection notification to the user's profile [extension]

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/FriendConnectionNotifier.scala [hunk 1]
```diff
@@ -45,9 +45,9 @@ class FriendConnectionNotifier @Inject() (
     val notificationF = elizaServiceClient.sendGlobalNotification( //push needed
       userIds = Set(friendUserId),
       title = s"You’re connected with ${respondingUser.firstName} ${respondingUser.lastName} on Kifi!",
-      body = s"Enjoy ${respondingUser.firstName}’s keeps in your search results and message ${respondingUser.firstName} directly. Find and invite more connections »",
+      body = s"Enjoy ${respondingUser.firstName}’s keeps in your search results and message ${respondingUser.firstName} directly.",
       linkText = "Invite more friends to kifi",
-      linkUrl = "https://www.kifi.com/friends/invite",
+      linkUrl = s"https://www.kifi.com/${respondingUser.username}",
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/FriendConnectionNotifier.scala]: @leogrim Is this the best way to do this now?

**Comment by andrewconner** [comment]
Let @leogrim answer that question, and then all good.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/FriendConnectionNotifier.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/FriendConnectionNotifier.scala]: I actually have a bug here :-1: need to do `username.value`

Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/FriendConnectionNotifier.scala]: Yep, you're good.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/FriendConnectionNotifier.scala [hunk 2]
```diff
@@ -45,9 +45,9 @@ class FriendConnectionNotifier @Inject() (
     val notificationF = elizaServiceClient.sendGlobalNotification( //push needed
       userIds = Set(friendUserId),
       title = s"You’re connected with ${respondingUser.firstName} ${respondingUser.lastName} on Kifi!",
-      body = s"Enjoy ${respondingUser.firstName}’s keeps in your search results and message ${respondingUser.firstName} directly. Find and invite more connections »",
+      body = s"Enjoy ${respondingUser.firstName}’s keeps in your search results and message ${respondingUser.firstName} directly.",
       linkText = "Invite more friends to kifi",
-      linkUrl = "https://www.kifi.com/friends/invite",
+      linkUrl = s"https://www.kifi.com/${respondingUser.username.value}",
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/FriendConnectionNotifier.scala:50]: re-adding @leogrim. Is this the way, or do we use something with handles?

Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/FriendConnectionNotifier.scala:50]: This is it.

**Comment by andrewconner** [comment]
Looks good then.

**Pull request merged**